File: apps/problems/classroom/boilerplate-full/function.cpp

#include <iostream>
#include <vector>
#include <string>

##USER_CODE_HERE##

int main() {
  int size_arr;
  std::cin >> size_arr;
  std::vector<int> arr(size_arr);
  for(int i = 0; i < size_arr; ++i) std::cin >> arr[i];
  int result = classroom(arr);
  std::cout << result << std::endl;
  return 0;
}
    

File: apps/problems/classroom/boilerplate-full/function.js
##USER_CODE_HERE##

const input = require('fs').readFileSync('/dev/problems/classroom/tests/inputs/##INPUT_FILE_INDEX##.txt', 'utf8').trim().split('\n').join(' ').split(' ');
const size_arr = parseInt(input.shift());
const arr = input.splice(0, size_arr).map(Number);
const result = classroom(arr);
console.log(result);
    